library/core/class.controller.php

```diff
@@ -1678,9 +1678,16 @@ public function RenderMaster() {
                if (StringBeginsWith($CssFile, 'http')) {
                   $this->Head->AddCss($CssFile, 'all', GetValue('AddVersion', $CssInfo, TRUE), $CssInfo['Options']);
                   continue;
-               } elseif(strpos($CssFile, '/') !== FALSE) {
-                  // A direct path to the file was given.
-                  $CssPaths = array(CombinePaths(array(PATH_ROOT, str_replace('/', DS, $CssFile))));
+               } elseif (strpos($CssFile, '/') !== FALSE) {
+                  $CssPaths = array();
+
+                  $AppFolder = $CssInfo['AppFolder'];
+                  if (empty($AppFolder)) {
+                     // A direct path to the file was given.
+                     $CssPaths[] = paths(PATH_ROOT, str_replace('/', DS, $CssFile));
+                  } else {
+                     $CssPaths[] = paths(PATH_APPLICATIONS, $AppFolder, 'design', $CssFile);
```

Comment:
Specifying "plugins/" is not compatible with also specifying an app folder since plugins are global.